rubocop.yml

```diff
@@ -90,6 +95,8 @@ Style/MutableConstant:
   Enabled: false
 Style/NumericPredicate:
   Enabled: false
+Style/OptionHash:
```

Comment:
Because we're generally on ruby 2.3 (and this cop won't do anything on non-supporting rubies). Declaring parameters as keywords is more explicit.